tests/system/python/packages/test_multiple_assets.py

```diff
@@ -78,8 +76,9 @@ def start_north(start_north_omf_as_a_service, fledge_url,
     asset_dict = {}
 
     no_of_services = 6
+    num_assets_per_service=(num_assets//no_of_services)
     for service_count in range(no_of_services):
-        for asst_count in range(PER_BENCHMARK_ASSET_COUNT):
+        for asst_count in range(num_assets_per_service):
             asset_name = ASSET_NAME + "-{}{}".format(service_count + 1, asst_count + 1)
             asset_dict[asset_name] = dp_list
```

Comment:
This code is not part of start_north and can be moved to add_benchmark

Reply:
Actually, it creates asset dictionary that passes to clear_pi_system_through_pi_web_api function for cleaning the PI Server.

Reply:
add an inline comment `Creates assets dictionary for PI server cleanup`